MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+        </listView:SfListView>
+    </StackLayout>
+</ContentPage.Content>
+{% endhighlight %}
+{% endtabs %}
+
+The `SwipeOffset` value should be positive for the left swiping of the listview item.
+ 
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void LeftSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], 200);
+}
+{% endhighlight%}
+{% endtabs %}
+
+The `SwipeOffset` value should be negative for the right-swiping of the listview item.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void RightSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], -150);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Programmatic Start and End swiping in listview](Images/swiping/maui-listview-programmatic-swiping.gif)
+
+## Events
+
+### SwipeStarted Event
+
+The `SfListView.SwipeStarting` event is raised when the swipe offset changes from its initial value. 
+
+The `SwipeStarting` event provides the following properties in their arguments:
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item. as its arguments. 
+ * `Direction`: Defines the swipe direction of the swiped item.
+ 
+The `SwipeStarting` event is used for the following use case:
+
+ * You can cancel the swipe action for a particular item by setting the `Cancel` property of the `SwipeStartingEventArgs`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfListView x:Name="listView" ItemsSource="{Binding InboxInfo}" 
+                     SwipeStarting="listView_SwipeStarting" />
+{% endhighlight %}
+{% highlight c# %}
+listView.SwipeStarting += ListView_SwipeStarting;
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ListView_SwipeStarting(object sender, SwipeStartedEventArgs e)
+{
+   if (e.Index == 1)
+      e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Swiping Event
+ 
+The `SfListView.Swiping` event is raised while swiping an item is in progress. This event is triggered with `SwipingEventArgs`.
+
+The `Swiping` event provides the following properties in their arguments:
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item as its arguments.
+ * `Direction`: Defines the swipe direction of the swiped item.
+ * `OffSet`: Defines the current swipe offset of the item being swiped.
+ * `Handled`: Defines that if it is `true`, current swipe offset value remains same for the swiped item until the `SwipeEnded` event is raised.
+
+The `Swiping` event used for the following use cases:
+
+* To maintain the current offset value for the swiped item till the `SwipeEnded` event gets called.
+* To hold the swipe view being swiping by setting the `Handled` property to `true` if swipe offset meet certain position.
```

Comment:
* Hold the swipe view from being swiped by setting the `Handled` property to `true` if the swipe offset meets a certain position.

Reply:
resolved